common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -3,19 +3,15 @@
 """
 
 # pylint: disable=abstract-method
-
-
 import collections
 import json
 import logging
 from datetime import datetime
 from functools import reduce
 
-from pkg_resources import resource_string
-
-import six
 from lxml import etree
 from opaque_keys.edx.keys import UsageKey
+from pkg_resources import resource_string
```

Comment:
Please ignore all pylint & iSort suggestions for core code, only keep your code changes and revert all the changes.